eng/Build.ps1

```diff
@@ -0,0 +1,286 @@
+#
```

Comment:
Feels really odd to have powershell in the repo after all this time, but it's simple enough and great to see the build.cmd go